qadence/engines/jax/differentiable_backend.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+from qadence.backend import Backend, ConvertedCircuit, ConvertedObservable
+from qadence.engines.differentiable_backend import (
+    DifferentiableBackend as DifferentiableBackendInterface,
+)
+from qadence.engines.jax.differentiable_expectation import DifferentiableExpectation
+from qadence.measurements import Measurements
+from qadence.mitigations import Mitigations
+from qadence.noise import Noise
+from qadence.types import ArrayLike, DiffMode, Endianness, Engine, ParamDictType
+
+
+class DifferentiableBackend(DifferentiableBackendInterface):
```

Comment:
Why not call it JaxDifferentiableBackend, or something? The import renames the thing with the same name, can be confusing :)

Reply:
i had it exactly like that :D but we do the same thing for the backends (meaning calling each individual backend class "Backend" instead of "{X}Backend"). i am open to both but i think @Roland-djee prefers it this way